tests/contrib/sensors/test_python_sensor.py

```diff
@@ -36,10 +35,6 @@ class PythonSensorTests(unittest.TestCase):
 
     def setUp(self):
         configuration.load_test_config()
-        self.dagbag = DagBag(
-            dag_folder=DEV_NULL,
```

Comment:
In this case, maybe https://github.com/apache/airflow/blob/bf7f48f2637cc0e898481915793f05531ec89229/tests/contrib/sensors/test_python_sensor.py#L31 can also be removed.

Reply:
Oh yes.